float32 x
float32 y
float32 theta
float32 linear_velocity
float32 angular_velocity
uint8 US_1
uint8 US_2
uint8 US_3
uint8 US_4
uint8 platform_state
uint8 left_motor_state
uint8 right_motor_state
